scripts/jenkins/qa_openstack.sh

```diff
@@ -41,6 +41,12 @@ fi
 
 mount -o remount,noatime,barrier=0 /
 
+# Start libvirtd and friends. Usually libvirtd is available and is
+# running, but can be that virtlogd was not automatically started.
+sudo service libvirtd status || sudo service libvirtd start
+if [[ -e /usr/lib/systemd/system/virtlogd.service ]] ; then
+    sudo service virtlogd status || sudo service virtlogd start
+fi
 
```

Comment:
We don't need/want that. It's a bug in libvirt and we just wait for a new image with ther fix. I'll monitor that.